app/views/govuk_publishing_components/audit/show.html.erb

```diff
@@ -10,21 +10,235 @@
   <%= render "components" %>
 <% end %>
 
-<% if @in_application or @other_applications %>
-  <%= render "govuk_publishing_components/components/tabs", {
-    tabs: [
-      {
-        id: "applications",
-        label: "Applications",
-        content: applications,
-      },
-      {
-        id: "components-gem",
-        label: "Components",
-        content: components,
-      }
-    ]
-  } %>
+        <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-with-summary-sections">
```

Comment:
This looks wrong... it's replacing some logic to display the tabs component with a hard coded accordion. I think we discussed this and this was an error relating to a merge?